stories/module-adsense-components.stories.js

```diff
@@ -189,6 +189,32 @@ const topEarningPagesArgs = {
 	limit: 5,
 };
 
+const topEarningPagesAdSenseReport = {
+	kind: 'adsense#report',
+	totalMatchedRows: '1',
+	headers: [
+		{ name: 'EARNINGS', type: 'METRIC_CURRENCY', currency: 'USD' },
+		{ name: 'PAGE_VIEWS_RPM', type: 'METRIC_CURRENCY', currency: 'USD' },
+		{ name: 'IMPRESSIONS', type: 'METRIC_TALLY' },
+	],
+	rows: [
+		[ '2103.23', '1.89', '5693093' ],
+	],
+	totals: [
+		'2103.23', '1.89', '5693093' ],
+	averages: [
+		'2103.23', null, '5693093',
+	],
+	startDate: '2020-08-15',
+	endDate: '2020-09-11',
+};
```

Comment:
Nitpick, but this is not a "correct" mock report for the data requested below:
* You're only requesting the `EARNINGS` metric below, as expected.
* This means the response must only include `EARNINGS` data. However the response here also includes data for two other metrics.